xev: add page (#6548)
tldr-pages/tldr: pages/common/xev.md

# xev

> Print contents of X events.
> More information: <https://gitlab.freedesktop.org/xorg/app/xev>.

- View all occuring X events:

`xev`

- Monitor the X events of the root window instead of creating a new one:

`xev --root`

- View the X events of a particular window:

> `xev -id {{Window_id}}`

Reviewer: ```suggestion
`xev -id {{window_id}}`
```

> `xev -id {{window_id}}`

Reviewer: One more inconsistency: Here you use one dash (`-id`), while the other examples use two (`--root`, `--event`). Could you normalize this to be consistent?

Reviewer 2: According to the [man page](https://gitlab.freedesktop.org/xorg/app/xev/-/blob/master/man/xev.man) it should be `-root` and `-event`.

Reviewer: Yes, it should. But that doesn't mean, it has to. I'm pretty sure, every X11 release ever has supported using two dashes. So it is a question of preference.

Author: It looks like `--id` does not work. I'm changing all to single `-`. 😄

Reviewer: That's weird.

> - View a category of X events (multiple flags can be chained):

Reviewer: First time reading this, I needed a minute to parse the sentence properly. Not sure, how to phrase it properly though. This would be ok, I think:

```suggestion
- View X events from a given category (multiple flags can be chained):
```

Author: Yea you're correct, that is easier to comprehend.

> `xev --event {{Event_category}}`

Reviewer: ```suggestion
`xev --event {{event_category}}`
```
